release notes/v0.48.0.md

```diff
@@ -0,0 +1,257 @@
+k6 v0.48.0 is here 🎉! This release includes:
+
+- Remote modules should be specified with full URLs.
+- Metrics names now need to conform to both OpenTelemetry (OTEL) and Prometheus name requirements.
+- A new `k6 new` subcommand to generate a new test script.
+- A new `k6/experimental/fs` module for file interactions.
+- k6 browser got a support for CPU and network throttling.
+
+## Breaking changes
+
+This release contains several breaking changes. Most of them should not heavily impact existing users, and have a clear process of migration.
+
+- [#3451](https://github.com/grafana/k6/pull/3451) In k6 [v0.25](https://github.com/grafana/k6/releases/tag/v0.25.0) module specifiers were allowed to be URLs with a scheme, and k6 prepending `https://` to anything that it wasn't recognizing was deprecated. From this version k6 will no longer try to do that all, please use full URLs if you want to load remote modules.
+- [#3448](https://github.com/grafana/k6/pull/3448) metric names now need to conform to both OpenTelemetry (OTEL) and Prometheus name requirements while still being limited to 128 ASCII characters (old k6 limit).
+- [#3350](https://github.com/grafana/k6/pull/3350) In k6 [v0.37](https://github.com/grafana/k6/releases/tag/v0.37.0), we deprecated a gRPC Invoke's parameter `headers`. This release removes the parameter. If you need to set headers, use a `metadata` parameter instead.
+- [#3389](https://github.com/grafana/k6/pull/3389) In k6 [v0.38](https://github.com/grafana/k6/releases/tag/v0.38.0) we deprecated `--logformat` introducing the better alternative `--log-format`. This release removes `--logformat` flag.
+- [#3390](https://github.com/grafana/k6/pull/3390) In k6 [v0.35](https://github.com/grafana/k6/releases/tag/v0.35.0) we deprecated all CSV output's CLI arguments having a not consistent case with other outputs. This release removes them.
+- [#3365](https://github.com/grafana/k6/pull/3365) In k6 [v0.41](https://github.com/grafana/k6/releases/tag/v0.41.0) we deprecated the `k6 convert` CLI command. This release deleted it, the better [har-to-k6](https://github.com/grafana/har-to-k6) alternative should be used.
```

Comment:
```suggestion
- [#3451](https://github.com/grafana/k6/pull/3451) In k6 [v0.25](https://github.com/grafana/k6/releases/tag/v0.25.0) module specifiers were allowed to be URLs with a scheme, and k6 prepending `https://` to anything that it wasn't recognizing was deprecated. From this version k6 will no longer try to do that all, please use full URLs if you want to load remote modules.
- [#3390](https://github.com/grafana/k6/pull/3390) In k6 [v0.35](https://github.com/grafana/k6/releases/tag/v0.35.0) we deprecated all CSV output's CLI arguments having a not consistent case with other outputs. This release removes them.
- [#3350](https://github.com/grafana/k6/pull/3350) In k6 [v0.37](https://github.com/grafana/k6/releases/tag/v0.37.0), we deprecated a gRPC Invoke's parameter `headers`. This release removes the parameter. If you need to set headers, use a `metadata` parameter instead.
- [#3389](https://github.com/grafana/k6/pull/3389) In k6 [v0.38](https://github.com/grafana/k6/releases/tag/v0.38.0) we deprecated `--logformat` introducing the better alternative `--log-format`. This release removes `--logformat` flag.
- [#3365](https://github.com/grafana/k6/pull/3365) In k6 [v0.41](https://github.com/grafana/k6/releases/tag/v0.41.0) we deprecated the `k6 convert` CLI command. This release deleted it, the better [har-to-k6](https://github.com/grafana/har-to-k6) alternative should be used.
- [#3448](https://github.com/grafana/k6/pull/3448) metric names now need to conform to both OpenTelemetry (OTEL) and Prometheus name requirements while still being limited to 128 ASCII characters (old k6 limit). Warnings about this started in [v0.45](https://github.com/grafana/k6/releases/tag/v0.45.0)
```

Order the breaking changes in since when they are depracated or had warnings.

Also add the version for the metric changes.

Reply:
I was tried to sort them by impact, for sure naively, but the idea was to try keep the most impactful on the top of the list :thinking:

Reply:
I've rewritten the section and tried to follow the structure: 

```
[issue] does, reference to the first announcement, how to mitigate
```

The order question is still discussable, and I could sort it by the versions, but IMO, it is more relevant to keep them sorted by impact.

Reply:
Adjusted :+1: